src/main/java/org/jenkinsci/plugins/p4/unshelve/UnshelveBuilder.java

```diff
@@ -28,19 +28,21 @@
 	private final String shelf;
 	private final String resolve;
 	private final boolean tidy;
+	private final boolean ignoreEmpty;
 
 	private static Logger logger = Logger.getLogger(UnshelveBuilder.class.getName());
 
 	@DataBoundConstructor
-	public UnshelveBuilder(String shelf, String resolve, boolean tidy) {
```

Comment:
You may need to deprecate the old constructor, just incase someone is using it in groovy.
```
@Deprecated
public UnshelveBuilder(String shelf, String resolve, boolean tidy) {
   this(shelf, resolve, tidy, false);
}
```

Reply:
Done.